Training24/CoinReturn.c

```diff
@@ -7,20 +7,19 @@
 // Program on Test2.2 branch.
 // ------------------------------------------------------------------------------------
 #include "CoinReturn.h"
+#include <stdbool.h>
 
-void CalculateChange (int paidAmount, int actualAmount, int* coinsReturned, int* coins, int numCoins) {
+bool CalculateChange (int paidAmount, int actualAmount, int* coinsReturned, int* coins, int numCoins) {
```

Comment:
`int* coinsReturned,`  instead of passing this to the method, better to create an array inside the method and return it.